src/main/java/com/openlattice/chronicle/services/ChronicleServiceImpl.java

```diff
@@ -179,7 +186,7 @@ public ApiClient load( Class<?> key ) throws Exception {
         titlePTID = edmApi.getPropertyTypeId( TITLE_FQN.getNamespace(), TITLE_FQN.getName() );
         fullNamePTID = edmApi.getPropertyTypeId( FULL_NAME_FQN.getNamespace(), FULL_NAME_FQN.getName() );
         recordTypePTID = edmApi.getPropertyTypeId( RECORD_TYPE_FQN.getNamespace(), RECORD_TYPE_FQN.getName() );
-        startDateTimePTID = edmApi.getPropertyTypeId( START_DATE_TIME.getNamespace(), START_DATE_TIME.getName() );
+        startDateTimePTID = edmApi.getPropertyTypeId( START_DATE_TIME_FQN.getNamespace(), START_DATE_TIME_FQN.getName() );
```

Comment:
Definitely not suggesting this for this PR, but I think you can make this much easier to maintain / adapt if you reduce the number of locations you need to modify code when updating something here. Example:

right now fqns/ids are being stored like this
```

FullQualifiedName PT_FQN_1 = blah1;
FullQualifiedName PT_FQN_2 = blah2;

...

private UUID ptId1;
private UUID ptId2;

...

ptId1 = edmApi.getPropertyTypeId( PT_FQN_1.getNamespace(), PT_FQN_1.getName() );
ptId2 = edmApi.getPropertyTypeId( PT_FQN_2.getNamespace(), PT_FQN_12getName() );
```
Given this pattern, 2 variables are created per property type and code must be updated (essentially copy/pasted) in 3 separate locations in order to add one. You could greatly cut down on the amount of code and also effort to add one by doing something like this instead:

```
FullQualifiedName PT_FQN_1 = blah1;
FullQualifiedName PT_FQN_2 = blah2;
FullQualifiedName[] PT_FQNS = new FullQualifiedName[] {
  PT_FQN_1,
  PT_FQN_2
}

...

Map<FullQualifiedName, UUID> propertyTypeIds = Maps.newHashMapWithExpectedSize( PT_FQNS.length );
for ( FullQualifiedName fqn : PT_FQNS ) {
  propertyTypeIds.put( fqn, edmApil.getPropertyTypeId( fqn.getNamespace(), fqn.getName() );
}
```

^ given this pattern, you would only need to add the FQN constant to use a new one, you could get the id from the map instead of storing a separate variable for it, and you wouldn't need to add a separate line to load and instantiate that variable. I tend to prefer this approach since there's less room for error of mis-assigning a property type id, less code, and less effort to add/remove/modify property types. (you can do the same thing for entity set ids too)

Reply:
Still need to validate, but: https://github.com/openlattice/chronicle-server/pull/103